modules/specifying-openshift-installer-log-levels.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-installations.adoc
+
+[id="specifying-openshift-installer-log-levels_{context}"]
+= Specifying {product-title} installer log levels
+
+By default, the {product-title} installer log level is set to `info`. If more detailed logging is required when diagnosing a failed {product-title} installation, you can increase the `openshift-install` log level to `debug` when starting the installation again.
+
+.Prerequisites
+
+* You have access to the installation host where `openshift-install` is being run.
```

Comment:
"being run" sounds odd to me..."is running"?

Reply:
I have changed this to '* You have access to the installation host."